teps/0096-pipelines-v1-api.md

```diff
@@ -174,11 +182,110 @@ V1 work should prioritize stabilizing existing beta features that support the [u
 If any gaps in current features are identified, they should be V1 blockers only if mission critical to common use cases.
 
 ## API Definition
+The current Tekton Pipelines API [definition](https://github.com/tektoncd/pipeline/blob/main/api_compatibility_policy.md#what-is-the-api) includes:
+
+- CRDs, including their spec and status fields
+- step ordering and naming of step containers within the Task status
+- labels propagated from PipelineRuns to TaskRuns and TaskRuns to Pods
+- the /workspace and /tekton/results [directories](https://github.com/tektoncd/pipeline/blob/main/docs/tasks.md#reserved-directories) inside containers started by Tekton
+- the interfaces of the images built by Tekton pipelines
+
+This policy should be updated to include Tekton metrics as part of the API. No other changes are proposed.
 
 ## Features Included
+### Documentation
+
+- Update the compatibility policy to reflect this TEP.
+- Provide a migration guide from v1beta1 API to v1 API.
+- Track documentation updates elsewhere (in a spreadsheet, milestone, or similar).
+
+### Production Readiness
+
+- Implement [TEP-0036: Start measuring performance](https://github.com/tektoncd/community/blob/main/teps/0036-start-measuring-tekton-pipelines-performance.md).
+- Implement [TEP-0073: Simplify metrics](https://github.com/tektoncd/community/blob/main/teps/0073-simplify-metrics.md).
+  - [TEP-0006: Tekton metrics](./0006-tekton-metrics.md) is out of scope because it discusses adding more granular metrics. More granular metrics may be added later.
+- Ensure metrics are accurate and mark all metrics as "alpha" or "stable". 
+
+### Stability
+
+- Track bug fixes and small backwards incompatible changes elsewhere (in a spreadsheet, milestone, or similar).
+- Implement conversion webhook for v1beta1 objects.
+
+#### CRD Stability Levels
+
+- The Task, TaskRun, Pipeline, and PipelineRun CRDs should be stabilized.
+- Because Custom Task is mission critical to our extensibility use case, Run should be upgraded to beta stability.
+  - See [Custom Tasks Beta](https://github.com/tektoncd/pipeline/issues/4313) for more information on work required to upgrade Custom Tasks to beta.
+- Keep the ClusterTask CRD at beta stability.
+  - ClusterTask functionality will likely be replaced by [remote resolution](./0060-remote-resource-resolution.md), but there's no plan
+  to replace this functionality at a beta level of stability before releasing a V1 API.
+  - We will decide whether to stabilize or remove ClusterTask pending community feedback on ClusterTask and remote resolution.
```

Comment:
ClusterTask are Task with a cluster scope. If Task are "stabilized", so are ClusterTask if we want to keep supporting them. The main question is do we want to deprecate them or not (and thus include them or not in `v1`).

Reply:
Based on discussion at yesterday's API WG, we aren't sure whether remote resolution will replace ClusterTask. I will work with @sbwsg and the community to figure this out. Because we're unsure of the state of ClusterTask, I don't want to propose stabilizing it. We can leave it at beta stability and leave this decision to after the v1 release, or I can update this TEP to reflect that a decision has not yet been made but will be made before v1.

Reply:
My point was more except from the scope, there is no feature specific to `ClusterTask`, so if we feel `Task` are stable and `v1` worthy, the only question we need to answer is if we want to have `Task` with `Cluster` scope or not. I think we are on the same page, it's just the wording that I find a bit confusing 😝

Reply:
Updated to link to issue on clusterTasks and note that this is still under discussion.